build.gradle

```diff
@@ -65,6 +65,16 @@ project.ext {
     rxAndroidVersion = "2.1.1"
     rxJavaVersion = "2.2.2"
     iconifyVersion = "2.2.2"
+    rxJavaRulesVersion = "1.3.3.0"
+    triangleLabelViewVersion = "1.1.2"
+
+    // Only used for free builds. This version should be updated regularly.
+    conscryptVersion = "2.4.0"
+    // Alternatively one can just use:
+    //  conscryptVersion = "latest.release"
+    // but it will mess up reproducible builds.
```

Comment:
Can you please remove this comment for the alternative? `latest.release` is not specific to conscrypt, so I do not think that it needs to be mentioned here. I prefer keeping explicit versions because that way, you can not get into the situation that the app suddenly stops working without changing the source code.